test/test_common/utility.cc

```diff
@@ -64,6 +65,11 @@ std::string TestUtility::bufferToString(const Buffer::Instance& buffer) {
   return output;
 }
 
+std::string TestUtility::bufferToHexString(const Buffer::Instance& buffer) {
```

Comment:
No, but it was useful in the test development. I might split this out into a separate PR.